recipe/run_test.sh

```diff
@@ -2,15 +2,12 @@
 
 set -e
 
+ls
 cd test
 ln -s ${GXX} g++
-cp ../xcrun .
-cp ../xcodebuild .
```

Comment:
Why? They are not in the same folder. `run_test.sh` is not run from the recipe directory